draft-hendrickson-privacypass-public-metadata.md

```diff
@@ -234,42 +225,46 @@ struct {
 The Token.nonce value is that which was sampled in {{Section 5.1 of PROTOCOL}}.
 If the Finalize function fails, the Client aborts the protocol.
 
+The Client will send this Token to Origins for redemption in the "token" HTTP
+authentication parameter as specified in {{Section 2.2 of AUTHSCHEME}}. 
+The Client also supplies its `extensions` value as an additional authentication
+parameter as specified in {{TBD}}.
+
 ## Token Verification
 
 Verifying a Token requires checking that Token.authenticator is a valid
-signature over the remainder of the token input using the Augmented Issuer Public Key.
+signature over the remainder of the token input with respect to the corresponding
+Extensions value `extensions` using the Augmented Issuer Public Key.
 
 ~~~
-pkM = AugmentPublicKey(pkI, Token.metadata)
+pkM = AugmentPublicKey(pkI, extensions)
 token_input = concat(0xDA7A, // Token type field is 2 bytes long
-                     int_to_bytes(len(Token.metadata), 2),
-                     Token.metadata,
                      Token.nonce,
                      Token.challenge_digest,
                      Token.token_key_id)
 token_authenticator_input =  concat("msg",
-    int_to_bytes(len(Token.metadata), 2),
-    Token.metadata,
+    int_to_bytes(len(extensions), 2),
```

Comment:
Shouldn't this be int_to_bytes(len(extensions), 4)?

Reply:
No, since the maximum size of Extensions (as [defined here](https://chris-wood.github.io/draft-wood-privacypass-extensible-token/draft-wood-privacypass-extensible-token.html#name-privatetoken-extensions-par)) is 2^16-1.

Reply:
Great. I guess it doesn't matter that PBRSA draft allows for bigger metadata then.

Reply:
Exactly!